musikcube.spec

```diff
@@ -0,0 +1,40 @@
+%define name musikcube
```

Comment:
It would be cool to support other RPM-based distributions in the future. These other distros, like openSUSE, may have slightly different package names for dependencies. In this case, would we just create another spec file? Like `musikcube_opensuse.spec` or something? Which, I guess, leads to the question: should this file be called `musikcube_fedora.spec`?

Reply:
I believe the spec can be used for any RPM-based distro, however it's worth testing. In any case the %{dist} macro will append the distro-specific tag, so we can distribute RPMs for multiple RPM-based distros.

Reply:
Looking into this more, we might need different specs, but that would only have to do with the dependencies for build and runtime, everything else can remain the same.